go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/rogpeppe/go-internal
 
-go 1.21
+go 1.21.0
```

Comment:
no no no :) keep it a "language minimal version" and not an "overly specific toolchain version" please
(just my 2 cents/crusade to keep/revert the go line to what it was before 1.21)

Reply:
resolved? without an answer?

Reply:
This really doesn't matter. Both mean almost exactly the same. If one dependency bumps us from 1.21 to 1.21.0, it's completely fine.